### PR TITLE
Fix not receiving props and prop owner names correctly on join

### LINKED ISSUE #4: Clientside owners don't update after player reconnects / joins to already spawned ents
New props - spawned after player already joined are received properly by the clientside,
but props that already were spawned before player connected will display N/A and won't have ownership on client (server's CPPIGetOwner will still return the correct player)

NADMOD.SendPropOwners() works properly (tho client might not receive it since it's instantly called on [PlayerInitialSpawn](https://wiki.garrysmod.com/page/GM/PlayerInitialSpawn)).

From my initial tests client receives only a small chunk of that containing weapons he's spawned with.
Sent IDs: 0-109 (last 10 being my props)
Received IDs: 83-94.
All that with few skipped of course due to IDs being unique.
 
I dislike networking to extend where I'm not even gonna start thinking of what's happening there, sorry @thegrb93 ;c

### PR DESCRIPTION
**Pull request by thegrb93**
Fixes: #4

### CODE REVIEW

File: lua/autorun/client/cl_nadmodpp.lua [hunk 1]
```diff
@@ -20,14 +20,11 @@ net.Receive("nadmod_propowners",function(len)
...
 	for i=1, net.ReadUInt(32) do
 		local id, str = net.ReadUInt(16), nameMap[net.ReadUInt(8)]
-		if id==0 then break end
 		if str == "-" then Props[id] = nil PropNames[id] = nil
 		elseif str == "W" then PropNames[id] = "World"
 		elseif str == "O" then PropNames[id] = "Ownerless"
 		else
 			Props[id] = str
-			local ply = player.GetBySteamID(str)
-			PropNames[id] = ply and ply:IsValid() and ply:Nick() or "N/A"
```
Comment (Nebual) [inline lua/autorun/client/cl_nadmodpp.lua:30]: I think if you keep this around, you'll get a cache of names that'll persist after the original owner has left. Its not perfect, but may as well :)

Or perhaps an
```
if ply and ply:IsValid() then
    PropNames[id] = ply:Nick()
end
```

**Review by Nebual — APPROVED** [review]

File: lua/autorun/client/cl_nadmodpp.lua (hunk 1 above)
Comment (thegrb93) [reply lua/autorun/client/cl_nadmodpp.lua:30]: Maybe I should just make the server send the player's name.

Comment (Nebual) [reply lua/autorun/client/cl_nadmodpp.lua:30]: Could have server keep a separate list of all players previously seen, and then send that once to players on their connect.

**Comment by thegrb93** [comment]
Will test tomorrow.

**Comment by thegrb93** [comment]
Yeah, this works.

**Pull request merged**